components/sensor/max31855.rst

```diff
@@ -30,16 +30,17 @@ Connect ``GND`` to ``GND``, ``VCC`` to ``3.3V`` and the other three ``MISO`` (or
         name: "Living Room Temperature"
         cs_pin: D2
         update_interval: 60s
+        reference_temperature:
```

Comment:
Does the normal user care about this value? If not, it should be removed here (should be minimal sample that the user can copy-paste)

Reply:
Good call. I will clean this up this evening.